### PR TITLE
Create our own draw and modify style functions

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: src/interaction.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+goog.provide('ngeo.interaction');
```
Comment (sbrunner) [inline src/interaction.js]: Shouldn't be in `ngeo.interaction.events`?

Comment (fredj) [reply src/interaction.js]: it's not related to events, only common function to all interactions

File: src/interaction/drawazimut.js [hunk 2]
```diff
@@ -87,8 +88,7 @@ ngeo.interaction.DrawAzimut = function(options) {
       useSpatialIndex: false,
       wrapX: false
     }),
-    style: options.style !== undefined ?
-      options.style : ol.interaction.Draw.getDefaultStyleFunction()
+    style: options.style !== undefined ? options.style : ngeo.interaction.getDefaultDrawStyleFunction()
```
Comment (gberaudo) [inline src/interaction/drawazimut.js]: You could simplify to
`style: options.style || ngeo.interaction.getDefaultDrawStyleFunction()`.

File: src/interaction/mobiledraw.js [hunk 3]
```diff
@@ -91,8 +92,7 @@ ngeo.interaction.MobileDraw = function(options) {
       useSpatialIndex: false,
       wrapX: options.wrapX ? options.wrapX : false
     }),
-    style: options.style ? options.style :
-      ol.interaction.Draw.getDefaultStyleFunction(),
+    style: options.style ? options.style : ngeo.interaction.getDefaultDrawStyleFunction(),
```
Comment (gberaudo) [inline src/interaction/mobiledraw.js]: Idem

File: src/interaction/modifycircle.js [hunk 4]
```diff
@@ -106,8 +106,7 @@ ngeo.interaction.ModifyCircle = function(options) {
       useSpatialIndex: false,
       wrapX: !!options.wrapX
     }),
-    style: options.style ? options.style :
-      ngeo.interaction.ModifyCircle.getDefaultStyleFunction(),
+    style: options.style ? options.style : ngeo.interaction.getDefaultModifyStyleFunction(),
```
Comment (gberaudo) [inline src/interaction/modifycircle.js]: Idem

**Pull request merged**